index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to Viable Groups</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class="navbar-black">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+            <div class="menu">
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                
+                <ul class="header-menu">
+                    <li class="header-menu-item"><a href="#work">English</a></li>
+                    <li class="header-menu-item"><a href="#about">My Page</a></li>
+                    <li class="header-menu-item"><a href="#contact">Login</a></li>
+                </ul>
+            </div> 
+        </nav> 
+        <nav class="navbar-white">
+            <div>
+                <img src="images/vg-logo.png" alt="VG Logo">
+            </div>
+            <ul class="header-white-menu">
+                <li class="header-white-menu-item"><a href="#about">About</a></li>
+                <li class="header-white-menu-item"><a href="#program">Program</a></li>
+                <li class="header-white-menu-item"><a href="#join">Join</a></li>
+                <li class="header-white-menu-item"><a href="#sponsor">Sponsor</a></li>
+                <li class="header-white-menu-item"><a href="#news">News</a></li>
+            </ul>
+            <button class="white-button" type="submit">CC Campaign</button>
+        </nav>  
+    </header>
+
+    <main>
+        <section class="headline">
+            <h1 class="headline-hello">Achievers world</h1>
+            <h2 class="headline-summit"> VIBRANT AFRICANS'<br>YOUTH SUMMIT 2022</h2>
+            <p class="headline-para">A global network of professional Software Developers 
+                aimed at sharing opportunities and creating a positive change 
+                with people from over 80 countries is taking place in Nigeria.
+            </p>
+            <p class="headline-venue">@Eko Hotel and Suites, Victoria Island, Lagos, Nigeria.</p>
+        </section>
+
+        <section class="program" id="program">
+            <h1 class="main-program">Main Program</h1>
```

Comment:
- [ ] Kindly set the same height for the cards of this section: 
<img width="772" alt="Screenshot 2022-04-13 at 20 17 32" src="https://user-images.githubusercontent.com/77038610/163225166-14548638-7114-4691-a070-93be78312022.png">